.github/workflows/pandas-tests.yaml

```diff
@@ -22,11 +22,14 @@ jobs:
   pandas-tests:
       # run the Pandas unit tests
       secrets: inherit
-      uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@branch-25.10
+      uses: rapidsai/shared-workflows/.github/workflows/custom-job.yaml@branch-25.10
       with:
-        matrix_filter: '[{"ARCH": "amd64", "PY_VER": "3.13", "CUDA_VER": "12.9.1", "LINUX_VER": "ubuntu24.04", "GPU": "l4", "DRIVER": "latest", "DEPENDENCIES": "newest"}]'
         build_type: nightly
         branch: ${{ inputs.branch }}
         date: ${{ inputs.date }}
         sha: ${{ inputs.sha }}
+        node_type: "gpu-l4-latest-1"
+        container_image: "rapidsai/citestwheel:25.10-cuda12.9.1-ubuntu24.04-py3.13"
```

Comment:
cc @jameslamb IIRC you had a PR to avoid hardcoding the cuda version in places. Should `rapidsai/ci-conda:25.10-latest` be a safe "default" image to use?

Reply:
> would this job still work if it started getting a CUDA 13 image?

Yeah this job just runs the pandas test suite with `cudf.pandas` enabled which should not be CUDA version dependent, so yes might as well change this to use the `-latest` image

Reply:
Got it, thanks @jameslamb @mroeschke . Updated the container image.